sdk/nodejs/Makefile

```diff
@@ -52,34 +54,43 @@ install_plugin:: build
 install:: install_package install_plugin
 
 unit_tests:: $(TEST_ALL_DEPS)
-	$(TESTSUITE_SKIPPED) --skip-windows unit-nodejs || \
-		yarn run nyc -x 'bin/tests/runtime/*ClosureCases*.js' -s mocha --timeout 120000 \
-			--exclude 'bin/tests/automation/**/*.spec.js' \
-			--exclude 'bin/tests/runtime/closureLoader.spec.js' \
-			'bin/tests/**/*.spec.js'
-	$(TESTSUITE_SKIPPED) --skip-windows unit-nodejs || \
-		yarn run mocha --timeout 120000 'bin/tests/runtime/closureLoader.spec.js'
-	$(TESTSUITE_SKIPPED) --skip-windows unit-with-mocks-nodejs || \
-		yarn run nyc -s mocha 'bin/tests_with_mocks/**/*.spec.js'
-
-auto_tests:: $(TEST_ALL_DEPS)
-	$(TESTSUITE_SKIPPED) --skip-windows auto-nodejs || yarn run nyc -s mocha --timeout 120000 'bin/tests/automation/**/*.spec.js'
-
-sxs_tests:: $(TEST_ALL_DEPS)
-	$(TESTSUITE_SKIPPED) --skip-windows sxs || ( \
-		pushd tests/sxs_ts_3.6    && yarn && tsc && popd && \
-		pushd tests/sxs_ts_latest && yarn && tsc && popd )
+	yarn run nyc -x 'bin/tests/runtime/*ClosureCases*.js' -s mocha --timeout 120000 \
+		--exclude 'bin/tests/automation/**/*.spec.js' \
+		--exclude 'bin/tests/runtime/closureLoader.spec.js' \
+		'bin/tests/**/*.spec.js'
+	yarn run mocha --timeout 120000 'bin/tests/runtime/closureLoader.spec.js'
+	yarn run nyc -s mocha 'bin/tests_with_mocks/**/*.spec.js'
+
+test_auto:: $(TEST_ALL_DEPS)
+	yarn run nyc -s mocha --timeout 120000 'bin/tests/automation/**/*.spec.js'
+
+TSC_SUPPORTED_VERSIONS = ~3.7.3 ^3 ^4
+
+version=$(subst sxs_test_,,$(word 1,$(subst !, ,$@)))
+sxs_test_%:
```

Comment:
These ones may be very slow good they're separate. There's something compiling a C++ node_gyp library as part of Node modules installation.

Reply:
updating the test target from Pulumi 1.6.0 fixed the node_gyp slowness and build error on Windows! 🎉